src/Stratis.Bitcoin.IntegrationTests/CoinViewTests.cs

```diff
@@ -9,16 +10,17 @@
 using Stratis.Bitcoin.Base;
 using Stratis.Bitcoin.Base.Deployments;
 using Stratis.Bitcoin.Consensus;
+using Stratis.Bitcoin.Consensus.CoinViews;
 using Stratis.Bitcoin.Consensus.Rules;
 using Stratis.Bitcoin.Features.Consensus;
-using Stratis.Bitcoin.Features.Consensus.CoinViews;
 using Stratis.Bitcoin.Features.Consensus.Rules.CommonRules;
 using Stratis.Bitcoin.IntegrationTests.Common;
 using Stratis.Bitcoin.IntegrationTests.Common.EnvironmentMockUpHelpers;
 using Stratis.Bitcoin.Tests.Common;
 using Stratis.Bitcoin.Utilities;
 using Xunit;
 using static NBitcoin.Transaction;
+using UInt256Comparer = Stratis.Bitcoin.Consensus.UInt256Comparer;
```

Comment:
If you remove the namespace at line 15, you can also remove this line - so won't need to alias UInt256Comparer.

Reply:
Fixed